database-migration/src/test/java/uk/ac/ebi/eva/dbmigration/mongodb/VariantData.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2016 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.dbmigration.mongodb;
+
+public class VariantData {
+
+    static final String VARIANT_WITHOUT_ANNOT = "{" +
```

Comment:
I'd prefer to have this json files as files more than strings but i'm not picky.